tests/unit/src/test/scala/tests/DebugProtocolSuite.scala

```diff
@@ -39,18 +47,14 @@ class DebugProtocolSuite extends BaseDapSuite("debug-protocol") {
       debugger <- server.startDebugging(
         "a",
         DebugSessionParamsDataKind.SCALA_MAIN_CLASS,
-        new ScalaMainClass(
-          "a.Main",
-          List("Bar").asJava,
-          List("-Dproperty=Foo").asJava
-        )
+        mainClass
       )
       _ <- debugger.initialize
       _ <- debugger.launch
       _ <- debugger.configurationDone
       _ <- debugger.shutdown
       output <- debugger.allOutput
-    } yield assertNoDiff(output, "FooBar")
+    } yield assertNoDiff(output, "FooBarFoo")
```

Comment:
I've tried to adapt this test to reflect my changes from #2123, but this test keeps failing for me even when I try to run it locally from the main branch and I can't quite figure out yet why. 🤔 

When I run `sbt unit/testOnly *DebugProtocolSuite* -- *.start*` I get the following:
```diff
=> Diff (- obtained, + expected)
-FooBar
+FooBarFoo
```

Do I need to do anything special with my local setup or the way those tests are run to get this test to pass?

I tried running them via `bin/test.sh "unit/testOnly *DebugProtocolSuite* -- *.start*"` and I can see in the logs bloop being set to `v1.4.4-13` rather than `1.4.4-17` which is what you changed it to in this PR.

Am I missing something obvious here? Any help or pointers would be much appreciated.

Reply:
no worries, we should not be probably using snapshot Bloop versions. I always creates additional confusion.